lib/conversations_client.ex

```diff
@@ -68,41 +77,96 @@ defmodule ExMicrosoftBot.Client.Conversations do
+        aid ->
+          "#{conversations_endpoint(service_url)}/#{conversation_id}/activities/#{aid}/members"
+      end
+
+    HTTPotion.get(api_endpoint, authed_req_options(api_endpoint))
+    |> deserialize_response(&Models.ChannelAccount.parse/1)
+  end
+
+  @doc """
+  This function takes a conversation ID and a member ID and returns a
+  ChannelAccount struct for that member of the conversation.
+
+  [API Reference](https://github.com/microsoft/botbuilder-js/blob/5b164105a2f289baaa7b89829e09ddbeda88bfc5/libraries/botframework-connector/src/connectorApi/operations/conversations.ts#L733-L749).
+  """
+  @spec get_member(String.t(), String.t(), String.t()) ::
+          {:ok, Models.ChannelAccount.t()} | Client.error_type()
+  def get_member(service_url, conversation_id, member_id) do
+    api_endpoint =
+      "#{conversations_endpoint(service_url)}/#{conversation_id}/members/#{member_id}"
+
+    HTTPotion.get(api_endpoint, authed_req_options(api_endpoint))
     |> deserialize_response(&Models.ChannelAccount.parse/1)
   end
 
   @doc """
   This method allows you to upload an attachment directly into a channels blob storage. [API Reference](https://docs.botframework.com/en-us/restapi/connector/#!/Conversations/Conversations_UploadAttachment)
   """
-  @spec upload_attachment(String.t, String.t, Models.AttachmentData.t) :: {:ok, Models.ResourceResponse.t} | Client.error_type
+  @spec upload_attachment(String.t(), String.t(), Models.AttachmentData.t()) ::
+          {:ok, Models.ResourceResponse.t()} | Client.error_type()
   def upload_attachment(service_url, conversation_id, %Models.AttachmentData{} = attachment) do
     api_endpoint = "#{conversations_endpoint(service_url)}/#{conversation_id}/attachments"
 
-    HTTPotion.post(api_endpoint, [body: Poison.encode!(attachment), headers: headers(TokenManager.get_token, api_endpoint)])
+    HTTPotion.post(
+      api_endpoint,
+      authed_req_options(api_endpoint, body: Poison.encode!(attachment))
+    )
     |> deserialize_response(&Models.ResourceResponse.parse/1)
   end
 
-  defp change_deserialized_response_to_ok({:ok, _}), do: :ok
-  defp change_deserialized_response_to_ok(resp), do: resp
+  @doc """
+  Updates an existing activity. The activity struct is expected to have an ID. [API Reference](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#update-activity)
+  """
+#  @spec update_activity(String.t(), String.t(), Models.Activity.t()) ::
+#          {:ok, Models.ResourceResponse.t()} | Client.error_type()
+#  def update_activity(service_url, conversation_id, %Models.Activity{id: activity_id} = activity) do
+#    api_endpoint =
+#      "#{conversations_endpoint(service_url)}/#{conversation_id}/activities/#{activity_id}"
+#
+#    api_endpoint
+#    |> HTTPotion.put(authed_req_options(api_endpoint, body: Poison.encode!(activity)))
+#    |> deserialize_response(&Models.ResourceResponse.parse/1)
+#  end
```

Comment:
Why commented?

Reply:
Is it the method that we changed once or it was added by new version of ex_microsoftbot?

Reply:
This method was added by the parent author and has a different signature then ours, so I've commented it not to mix them up.
I didn't decide yet if I will change our code to fit this signature or left our method.

Reply:
my proposal for this issue:
https://github.com/PagerDuty/ex_microsoftbot/pull/9#discussion_r587541944